plugins/hydra_nevergrad_sweeper/setup.py

```diff
@@ -20,10 +20,6 @@
             "Programming Language :: Python :: 3.8",
             "Operating System :: OS Independent",
         ],
-        install_requires=[
-            "hydra-core>=1.0.0rc1",
-            # TODO move to v0.4.0 when ready (before merge!)
-            "nevergrad@git+git://github.com/facebookresearch/nevergrad.git@master#egg=isort",
-        ],
+        install_requires=["hydra-core>=1.0.0rc1", "nevergrad>=0.4.0"],
```

Comment:
I think for now to pin plugins to a major version of Hydra.
```
hydra-core>=1.0.*
```

Will try it out once the dust settles.